git-redate

```diff
@@ -73,11 +74,17 @@ while [[ $# -ge 1 ]] ; do
             echo "  -c| --commits N Process last N commits, default=5."
             echo "  -d| --debug     Enable debug output."
             echo "  -h| --help      Show this help message."
+            echo "  -v| --version   Show the version (${VERSION}) of git-redate"
```

Comment:
Please could you change this to `Show the version of git-redate"` ? Else the `-v` parameter wouldn't make sense, could be directly part of the help message.